doc/source/_toc.yml

```diff
@@ -62,20 +62,21 @@ parts:
           - file: train/key-concepts
             title: Key Concepts
           - file: train/getting-started-pytorch
-            title: PyTorch
+            title: "PyTorch Guide"
```

Comment:
Are the quotes needed?

Reply:
Good catch - I'll remove them. I had tried different strings and in one case I used `:`, which YAML interprets.